adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -21,6 +21,13 @@ table below.
 
 == Usage example
 
+The example below demonstrates the usage of this extension.
```

Comment:
Should we mention here that this is using queue recording/record and replay in the example specifically? Might make more sense if we're also going to add an explicit API example at some point.

Reply:
done.